files.sh

```diff
@@ -1,20 +1,26 @@
 # Autocomplete of files
 
 fn nash_complete_paths(parts, line, pos) {
-	var partsz <= len($parts)
-	var last <= -expr $partsz - 1
-	var last <= trim($last)
-	var lastpart <= echo -n $parts[$last] | sed $sedArgs "s#^~#"+$HOME+"#g"
-
+	var choice = ""
 	var dir = ""
-	var fname = ""
 	var dirpath = ""
+	var findArgs = ()
+	var fname = ""
+	var last = ""
+	var lastpart = ""
+	var partsz = ""
+	var status = ""
+
+	partsz           <= len($parts)
+	last, status     <= expr $partsz - 1
```

Comment:
if the objective is to ignore the status and just dont crash you can use **_** to make the intention explicit (I was looking to where you use status but it seems that you dont).

Reply:
Yes, your right ;)

Reply:
done! fix that!